### ISSUE TITLE
After doing a "Try" the form data gets resetted.

### ISSUE THREAD

**Issue by Uzlopak** [mrin9/RapiDoc#306]
Is there a way to keep the form data after doing a rest call?

**Comment by snazzyfox** [comment]
Running into this as well. Looks to me like a bug introduced in 8.2.0. My workaround has been using the previous version (8.1.2) instead.

I also found that the form clears itself not only when you click the "try" button, it also does so when you switch between the "model" and "example" tabs.

**Comment by mrin9** [comment]
This behavior has been added as a part of #281 
it was necessary to reset the form to indicate our underlying reactive framework to trigger reactivity.

We are waiting on for our main frameworks `lit-html` and `lit-element` for an upgrade to address this issue properly. 

> `lit-html` and `lit-element` are getting ready for a major upgrade and we have our eyes on them

**Comment by BradHolmes** [comment]
Hi,

Any update on the timeline for this fix?  

Thanks!

**Comment by adamwhitakerwilson** [comment]
This would be great to have implemented, are there any workarounds for now that anyone can suggest?

**Comment by Uzlopak** [comment]
@mrin9 did it get fixed?

**Comment by Uzlopak** [comment]
Ah, I see the commit. Nice. Waiting eagerly for the release.

**Comment by mrin9** [comment]
let me know if it worked for your use case ... you can pick the latest build from `dist/rapidoc-min.js` to test it

**Comment by BradHolmes** [comment]
It works great for regular fields, but array fields still reset.  In addition, there are some odd bugs in trying to set a *default* for array fields with multiple values.  For one, the fields are smushed together.  I may have my specification wrong:

```
        - name: my_param
          in: path
          required: true
          schema:
            type: array
            items:
              type: string
              default:
                - whiz
                - bang
                - foo
                - bar
```

I'm happy to create a new issue, if that helps.

**Comment by mrin9** [comment]
@BradHolmes please create a new issue, we do not get notification on closed issues.

Some of the things you described are being worked on, but do provide your version of issues. 

it helps us to distribute work if you create one issue for one bug, also a complete (not partial) but minimal spec that highlights the problem gets a better chance to be picked up

**Comment by schmurfy** [comment]
@BradHolmes was it solved ? I am asking but I still have that behaviour with the latest release
I looked but could not find any other issue opened or closed related to this.

**Comment by BradHolmes** [comment]
Yes, sorry I didn't post back here with the new issue.  It is https://github.com/mrin9/RapiDoc/issues/353
